Fix os_cmd/Linux/ossdetect/ossdetect.c compile problem with Linux kernel 4.14, need to include sys/sysmacros.h

--- a/oss-v4.2-build2017-src-gpl/os_cmd/Linux/ossdetect/ossdetect.c
+++ b/oss-v4.2-build2017-src-gpl/os_cmd/Linux/ossdetect/ossdetect.c
@@ -23,6 +23,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/dir.h>
+#include <sys/sysmacros.h>
 
 #define PCI_PASS	0
 #define USB_PASS	1
